### PR TITLE
chore: fix spacing of js_library docstrings

### PR DESCRIPTION
**Pull request by gregmagolan**
These docstrings indents are off

### CODE REVIEW

**Comment by alexeagle** [comment]
it's fixed in our stardoc actually...
https://github.com/bazelbuild/bazel/pull/13403
https://github.com/aspect-build/stardoc/commit/34d1bdbf6aa58f676ff4b69d77ff434672ba49ac
😁

**Comment by gregmagolan** [comment]
The docsite I keep having to manually fix things after slurping & generating 😢

File: js/private/js_library.bzl [hunk 1]
```diff
@@ -46,45 +46,45 @@ _ATTRS = {
...
-        ```
-        js_library(
-            name = "js_lib",
-            declarations = ["index.js"],
-        )
-        ```
-        """,
+```
+js_library(
+    name = "js_lib",
```
Comment (jbedard) [inline js/private/js_library.bzl:70]: I think these lines will still be broken if I'm reading this right... :(

**Review by jbedard — APPROVED** [review]

File: js/private/js_library.bzl (hunk 1 above)
Comment (gregmagolan) [reply js/private/js_library.bzl:70]: They should be good. The code block is left aligned.

Comment (jbedard) [reply js/private/js_library.bzl:70]: I see no new lines or `<br>` in the generated html though...?

**Comment by gregmagolan** [comment]
Ohhh.. I see what the root cause is on the docsite. Some of these docstrings end up with mixed indentations since they are build up from string partials with format 😵‍💫

File: js/private/js_library.bzl [hunk 2]
```diff
@@ -46,45 +46,45 @@ _ATTRS = {
     "srcs": attr.label_list(
         doc = """Source files that are included in this library.
```
Comment (alexeagle) [inline js/private/js_library.bzl:47]: nit: when I'm paranoid and de-dent docs to the file level, I start with backslash-newline so the first line is also at column 0
```
          """\
First line goes here
```

**Pull request merged**